5-causation/03-matching/notes.qmd

```diff
@@ -239,7 +238,6 @@ head(grad_job, n = 20)
 
 We conduct nearest-neighbor matching.  Since there are 5 treated units and 15 controls, we end up forming five matched pairs and discarding the 10 leftover controls (if we had started with more treated units, we would have discarded some of them instead).  
 
-
```

Comment:
I think there's an opportunity to polish up the tables to clarify what is getting matched to what. I'm imagining two data frames, one adjusted, one unadjusted to mesh better with the Love plot. I'll work on this either later this afternoon or tomorrow. Let's not let this hold up the publication, though, I can republish midweekend.